bids/variables/io.py

```diff
@@ -82,6 +82,24 @@ def load_variables(layout, types=None, levels=None, skip_empty=True,
     return dataset
 
 
+def _get_timing_info(img_md):
+    if 'RepetitionTime' in img_md:
+        tr = img_md['RepetitionTime']
+        if 'DelayTime' in img_md:
+            ta = tr - img_md['DelayTime']
+        elif 'SliceTiming' in img_md:
+            slicetimes = sorted(img_md['SliceTiming'])
+            # a, b ... z
+            # z = final slice onset, b - a = slice duration
+            ta = slicetimes[-1] + slicetimes[1] - slicetimes[0]
```

Comment:
should there be some kind of precision check. as a floating point number this can differ from tr, but may not be different.